src/constants/tokenLists/scroll-alpha.tokenlist.json

```diff
@@ -19,10 +19,10 @@
       "logoURI": "https://assets.coingecko.com/coins/images/6319/thumb/USD_Coin_icon.png?1547042389",
       "chainId": 534354,
       "address": "0x38BA9a208F34Ddc9332f6DfC0E9d567f098958A4",
```

Comment:
I don't think this is right, so we probably just want to remove it to avoid confusion? Or I guess distinguish between chainIds?